java-api-core/src/com/kinvey/java/model/KinveyPullResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.kinvey.java.model;
+
+
+public class KinveyPullResponse extends KinveyAbstractResponse {
```

Comment:
Out of curiosity, what is the reason for moving the pull response out of `android` and into `java-api-core`?

Reply:
`pull` from `BaseDataStore` has the same response as `pull` from the DataStore, so I think It's logical to have one response class for both methods.